lambda/package.json

```diff
@@ -14,16 +14,17 @@
     "babel-core": "^6.24.1",
     "babel-preset-env": "^1.4.0",
     "lambda-local": "ashiina/lambda-local#316f6c743f6305385655b00db7d8349dac174c1c",
-    "node-riffraff-artefact": "^2.0.1"
+    "node-riffraff-artefact": "^2.0.1",
+    "google-spreadsheet": "^2.0.4"
   },
   "scripts": {
     "riffraff-artefact": "./node_modules/.bin/riffraff-artefact",
     "dist" : "yarn install --production --modules-folder dist/node_modules",
     "compile": "babel src -d dist",
     "riffraff": "VERBOSE=true ARTEFACT_PATH=$PWD yarn riffraff-artefact",
     "local":"yarn run:index && yarn run:hello",
-    "run:index": "lambda-local -l dist/index.js -e event.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 10 -E {\\\"Stage\\\":\\\"CODE\\\"}",
-    "run:hello": "lambda-local -l dist/index.js -e event.json -h handler -P ~/.aws/credentials -p membership -r eu-west-1 -t 10 -E {\\\"Stage\\\":\\\"CODE\\\"}",
+    "run:index": "lambda-local -l dist/index.js -e event.json -h handler -P ~/.aws/credentials -p composer -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"CODE\\\"}",
+    "run:hello": "lambda-local -l dist/index.js -e event.json -h handler -P ~/.aws/credentials -p composer -r eu-west-1 -t 30 -E {\\\"Stage\\\":\\\"CODE\\\"}",
```

Comment:
`run:hello`? should that be `run:google` or `run:config`? Or just removed because it looks the same as above right?

Reply:
ah that's some hangover from Alex's boilerplate. Will get rid